### PR TITLE
CB-498 - StackCreationTest E2E test: AWS stack creation working

### PR DESCRIPTION
**Pull request by attilapalfi92**

### CODE REVIEW

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/mock/ClusterTemplateTest.java [hunk 1]
```diff
@@ -184,7 +184,7 @@ public void testCreateInvalidShortNameClusterTemplate(TestContext testContext) {
     public void testCreateAgainClusterTemplate(TestContext testContext) {
         testContext.given("environment", EnvironmentEntity.class).withRegions(VALID_REGION).withLocation(LONDON)
                 .when(Environment::post)
-                .given("placementSettings", PlacementSettingsEntity.class).withRegion(EUROPE)
+                .given("placementSettings", PlacementSettingsEntity.class).withRegion(MockCloudProvider.EUROPE)
```
Comment (topolyai5) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/mock/ClusterTemplateTest.java]: is it mock only?

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/mock/BlueprintTest.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.sequenceiq.it.cloudbreak.newway.testcase;
+package com.sequenceiq.it.cloudbreak.newway.testcase.mock;
```
Comment (topolyai5) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/mock/BlueprintTest.java]: is it run on circle? Is the yaml pattern include the subpackage?

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/mock/AbstractIntegrationTest.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package com.sequenceiq.it.cloudbreak.newway.testcase;
+package com.sequenceiq.it.cloudbreak.newway.testcase.mock;
```
[the remark this answers is not in the document]
Comment (attilapalfi92) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/mock/AbstractIntegrationTest.java]: true

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/mock/ClusterTemplateTest.java (hunk 1 above)
Comment (attilapalfi92) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/mock/ClusterTemplateTest.java]: yes

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/mock/BlueprintTest.java (hunk 2 above)
Comment (attilapalfi92) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/mock/BlueprintTest.java]: I update it

**Pull request merged**